app/views/organizations/_form.html.erb

```diff
@@ -14,6 +14,17 @@
     </label>
   </div>
 
+  <% if organization.persisted? %>
+    <hr />
+
+    <h3>Item information</h3>
+    <%= form.hidden_field 'normalization_steps[0][destination_tag]', maxlength: 3, value: organization.normalization_steps.values.dig(0, 'destination_tag') || 998 %>
```

Comment:
Any particular reason we're making this a hidden field?  Maybe it could be a disabled text input?  🤷

Reply:
@mejackreed and I went back and forth on what to do with these fields and whether to add some abstraction in the form (that we then map back to MARC fields in the app) or just expose the raw MARC mappings and call it good enough for now.

So, 🤷‍♂️ ? I'm happy to go either way.

Reply:
The 🤷s have it